### ISSUE TITLE
What's the status of this project?

### ISSUE THREAD

**Issue by benwiley4000** [DOL-Translations/drill-land#1]
Hi there, I saw this project is pretty new, but I'm excited about it since I've started to play Drill Land and I'm realizing a lot of the menu text is over my head. I tried out the manual patch but it seems like most of the content hasn't been translated yet. Are you planning to translate the rest?

Cheers

**Comment by NewGBAXL** [comment]
Yes, we still just started and not many changes have been made yet.  Many of our team are just getting

**Comment by NewGBAXL** [comment]
Absolutely! This translation is just starting out and many of our team members are using this as a nice entry point for learning how to make a translation. Almost all of the text is graphical and in a standard format, so while it is mostly a straightforward process, it will take some time. In the meantime, I recommend supporting the series with the remaster, as those sales will determine if new original games will be released in the future.

**Comment by NewGBAXL** [comment]
New release very soon™️
